HexCtrl/src/CHexCtrl.cpp

```diff
@@ -385,6 +385,13 @@ bool CHexCtrl::Create(const HEXCREATE& hcs)
 	m_pDlgSearch->Create(IDD_HEXCTRL_SEARCH, this, this);
 	m_pDlgGoTo->Create(IDD_HEXCTRL_GOTO, this, this);
 	m_pBookmarks->Attach(this);
+
+	//Determine current user locale specific date format. Default to UK/European if unable to determine
+	//See: https://docs.microsoft.com/en-gb/windows/win32/intl/locale-idate
+	if (!GetLocaleInfoEx(LOCALE_NAME_USER_DEFAULT, LOCALE_SSHORTDATE | LOCALE_RETURN_NUMBER,
+		reinterpret_cast<LPWSTR>(&m_dwDateFormat), sizeof(m_dwDateFormat)))

```

Comment:
The `LOCALE_RETURN_NUMBER` can be combined with any other constant having a name that begins with "LOCALE_I" ([MSDN](https://docs.microsoft.com/en-us/windows/win32/intl/locale-return-constants)).

`LOCALE_SSHORTDATE` isn't correct here, I think `LOCALE_IDATE` should be used.

Reply:
This is exactly the same code that is currently in CHexDlgDataInterp but I agree. LOCALE_IDATE seems more correct.